Detectors/ITSMFT/ITS/tracking/src/Tracker.cxx

```diff
@@ -444,6 +444,8 @@ void Tracker::traverseCellsTree(const int currentCellId, const int currentLayerI
 void Tracker::computeRoadsMClabels(const ROframe& event)
 {
   /// Moore's Voting Algorithm
+  if (!event.hasMCinformation())
+    return;
```

Comment:
Even though I personally don't like it, the coding guidelines require braces here.

Reply:
I really hoped no one realised (since it is not enforced :P ). I still have to update this PR with another modification, I will then put the braces.